src/pyproject.toml.m4

```diff
@@ -1,7 +1,6 @@
 [build-system]
 # Minimum requirements for the build system to execute.
 requires = [
-    "sage_setup[autogen]",
```

Comment:
@fbissey Had to remove this clever solution from one of our changes not so long ago. This ran into a defect that was apparently solved in https://github.com/pypa/pip/pull/12002 but hasn't made its way into released versions yet

Reply:
That comment never reached me since github does not know me by that name :) I must say I do not exactly remember it, I am chasing way too many hares to keep that level of details into memory.

Reply:
Ah, sorry, my mistake!